src/class/methods.rs

```diff
@@ -29,16 +30,22 @@ pub enum PyMethodType {
     PyCFunctionWithKeywords(PyCFunctionWithKeywords),
 }
 
-// These two newtype structs serve no purpose other than wrapping the raw ffi types (which are
-// function pointers) - because function pointers aren't allowed in const fn, but types wrapping
-// them are!
+// These newtype structs serve no purpose other than wrapping which are function pointers - because
```

Comment:
?
```suggestion
// These newtype structs serve no purpose other than wrapping (raw ffi) function pointers - because
```

Reply:
Actually the `ClassAttributeFactory` is a Rust function pointer that we use internally and not for ffi :)